README.md

```diff
@@ -183,6 +190,8 @@ Text embeddings allow deep learning to be effective on smaller datasets. These a
 
 The most popular names in word embeddings are word2vec by Google (Mikolov) and GloVe by Stanford (Pennington, Socher and Manning). fastText seems to be a fairly popular for multi-lingual sub-word embeddings. 
 
+[Back to Top](#contents)
```

Comment:
Some are in the bottom of the section and some are right below the section title.